url_handlers.go

```diff
@@ -96,6 +96,24 @@ func WaitListHandler(w http.ResponseWriter, r *http.Request) {
   RenderTemplate(w, "assets/templates/waitlist.html.tmpl", nil)
 }
 
+func AssignedBuzzerHandler(w http.ResponseWriter, r *http.Request) {
+    if !IsUserLoggedIn(GetSession(w, r)) {
+      http.Redirect(w, r, "/login", 302)
+      return
+    }
+
+    if r.Method == "POST" {
+        active_party_id = r.FormValue("active_party_id")
+
+        var activeparty ActiveParty
+        db.First(&activeparty, active_party_id)
+
+        if activeparty.BuzzerID != nil:
+            return true
```

Comment:
You shouldn't be returning a value here, you should be rendering JSON. The `SampleJSONHandler` also shows how that works